charts/helm-chart/kubernetes-dashboard/ci/extra-manifests-values.yaml

```diff
@@ -1,21 +0,0 @@
-# Copyright 2017 The Kubernetes Authors.
```

Comment:
All the `ci/.*-values.yaml` files are tested by chart-testing, AND can act as valid use cases: I think we should keep/adapt them.

Reply:
We definitely can. We just didn't have time to do this. We were mostly focused on making it work.